requirements.txt

```diff
@@ -1,6 +1,7 @@
 colorama==0.3.9
 urllib3[secure]>=1.26.5
-prettytable==0.7.2
+PrettyTable<=1.0.1; python_version < '3.6'
```

Comment:
Do we need to handle version < 3.6? From https://github.com/ARMmbed/mbed-os/commit/b8388cd76e5c9df57e12bf67c7a41a47a73ade75:

> A few libraries we depend on no longer support old versions of Python
like 3.5 and 3.6. Remove testing for these old versions.

Reply:
We haven't removed python 2 support, yet, so this covers there. Old python 3 doesn't necessarily need supporting, but there is no need to block it by requiring a prettytable version that has dropped support for that older python version.

Reply:
Fair enough